docs/contribute.rst

```diff
@@ -22,9 +22,9 @@ another, so you can use different packages (and package versions) for each.
 Please note that Python 3.6+ is required for Pelican development.
 
 *(Optional)* If you prefer to install Poetry once for use with multiple projects,
-you can install it via::
+you can install it via (readmore at `Poetry Installation <https://python-poetry.org/docs/master/#installation>`_)::
```

Comment:
Would you please remove the changes to this line, and instead modify the line above to contain the relevant link? Afterward, it would look like:

```rst
*(Optional)* If you prefer to `install Poetry <https://python-poetry.org/docs/master/#installation>`_ once for use with multiple projects,
```

Reply:
Thanks for review, sorry i am still learning English. I will use your suggestions.